contexts/retail-coupon/vocab/index.html

```diff
@@ -0,0 +1,410 @@
+            <td>A string code that may help disambiguate which item the discount refers to. Commonly empty string (<code>""</code>).</td>
+          </tr>
+          <tr>
+            <td id="store_coupon">
+              <code>store_coupon</code>
+            </td>
+            <td>Store Coupon</td>
+            <td>String value that identifies whether a coupon is a store coupon. Usage not documented in public materials. Commonly <code>"1"</code>.</td>
+          </tr>
+          <tr>
+            <td id="donot_multiply_flag">
+              <code>donot_multiply_flag</code>
+            </td>
+            <td>Do Not Multiply Flag</td>
+            <td>String value that controls handling of multiple discounts. Usage not documented in public materials. Commonly empty string (<code>""</code>).</td>
+          </tr>
+      </table>
+    </section>
+
+    <section>
+      <h2>Example Credential</h2>
+      <pre class="example">
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://contexts.vcplayground.org/examples/retail-coupon/v1.json"
+    ],
+    "type": ["VerifiableCredential", "ClippedCouponCredential"],
+    "issuer": {
+        "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+        "name": "Big Retail Store",
+        "url": "https://example.com",
+        "image": "https://playground.chapi.io/examples/retail-coupon/logo.png"
+    },
+    "name": "Digital coupon: 50% off on disco balls, small or large",
+    "issuanceDate": "2023-06-04T00:00:00Z",
+    "expirationDate": "2023-08-01T08:00:00Z",
+    "image": "https://playground.chapi.io/examples/retail-coupon/image.png",
+    "credentialSubject": {
+        "id": "did:key:123",
+        "clippedCoupon": {
+        "type": "CouponOffer",
+        "base_gs1": "8112010031493140181",
+        "brand_id": "Party Palace",
+        "description": "50% off on disco balls, small or large",
+        "campaign_start_time": "06/01/2023",
+        "campaign_end_time": "06/30/2023",
+        "redemption_start_time": "06/01/2023",
+        "redemption_end_time": "07/31/2023",
+        "total_circulation": "100",
+        "primary_purchase_save_value": "1",
+        "primary_purchase_requirements": "1",
+        "primary_purchase_req_code": "1",
+        "primary_purchase_gtins": ["294239749273", "239084320923"],
+        "additional_purchase_rules_code": "",
+        "second_purchase_requirements": "",
+        "second_purchase_gs1_company_prefix": "",
+        "second_purchase_req_code": "",
+        "second_purchase_gtins": [""],
+        "third_purchase_requirements": "",
+        "third_purchase_gs1_company_prefix": "",
+        "third_purchase_req_code": "",
+        "third_purchase_gtins": [""],
+        "gln": "",
+        "save_value_code": "1",
+        "applies_to_which_item": "",
+        "store_coupon": "1",
+        "donot_multiply_flag": ""
+        }
+    }
+}
+              
+        </pre
+      >
```

Comment:
I see a handful of these closing angle brackets on another line.  Is that needed here?
```suggestion
      </pre>
```

Reply:
Sometimes the auto-formatters do that in HTML when they're trying to preserve some intent about whitespace. Looks like it was somewhat confused this time.